chef/cache/src/main/java/org/jclouds/karaf/chef/cache/Activator.java

```diff
@@ -61,7 +65,9 @@ public void start(BundleContext context) throws Exception {
         cacheProviderRegistration = context.registerService(CacheProvider.class.getName(), cacheProvider, new Hashtable<String, Object>());
 
 
-        chefServiceTracker = CacheUtils.createServiceCacheTracker(context, ChefService.class, chefCacheManager);
+        chefServiceTracker = CacheUtils.createServiceCacheTracker(context, new TypeToken<ApiContext<ChefApi>>() {
+           private static final long serialVersionUID = 1L;
```

Comment:
The chef-cache project does not have the chef-core as a dependency, and I wanted to avoid adding it just to declare the `TypeToken`. Are you OK leaving this as-is?